lib/ecosystems/enrich_cyclonedx.go

```diff
@@ -63,7 +64,7 @@ func enrichCDXLicense(component cdx.Component, packageData packages.Package) cdx
 	return component
 }
 
-func enrichExternalReference(component cdx.Component, packageData packages.Package, url *string, refType cdx.ExternalReferenceType) cdx.Component {
+func enrichExternalReference(component cdx.Component, _ packages.Package, url *string, refType cdx.ExternalReferenceType) cdx.Component {
```

Comment:
Change to appease the linter.

Reply:
While we're at it, would it make sense to remove the `packages.Package` param altogether and possibly extract the `enrichExternalReference` to the cdx utils? could be reused in the future for enrichers other than `ecosystems`

Reply:
While we're at it, would it make sense to remove the `packages.Package` param altogether and possibly extract the `enrichExternalReference` to the cdx utils? could be reused in the future for enrichers other than ecosystems

Reply:
i'll change this in a later PR if that's ok. IMHO we should change all the enrichers anyway, to receive a pointer to a component and mutate; we could do this all in a refactoring PR.